packages/core/src/browser/preferences/preference-proxy.ts

```diff
@@ -28,8 +28,9 @@ export interface PreferenceEventEmitter<T> {
 export type PreferenceProxy<T> = Readonly<T> & Disposable & PreferenceEventEmitter<T>;
 export function createPreferenceProxy<T extends Configuration>(preferences: PreferenceService, schema: PreferenceSchema): PreferenceProxy<T> {
     const configuration = createConfiguration<T>(schema);
-    const validate = (name: string, value: any) => validatePreference(schema, { [name]: value });
-
+    const ajv = new Ajv();
+    const validate = ajv.compile(schema);
```

Comment:
it should be: 
```ts
const ajv = new Ajv();
const validatePreference = ajv.compile(schema);
const validate = (name: string, value: any) => validatePreference({ [name]: value });
```

otherwise new validate does not match signature of validate before